apps/gp2-frontend/src/__tests__/App.test.tsx

```diff
@@ -37,7 +37,7 @@ it('changes routing for logged in users', async () => {
   expect(container).toHaveTextContent(/Signin/i);
   rerender(
     <authTestUtils.Auth0ProviderGP2>
-      <authTestUtils.LoggedInGP2 user={{}}>
+      <authTestUtils.LoggedInGP2 user={{}} onboarded={true}>
```

Comment:
The default in packages/react-components/src/auth-test-utils.tsx is for the user to be onboarded. So we shouldn't need to set this property everywhere